2.process-data/split_training_test.py

```diff
@@ -0,0 +1,47 @@
+import os
+import numpy as np
+import pandas as pd
+
+from sklearn.model_selection import train_test_split
+
+np.random.seed(123)
+
+# Define paths
+data_dir = os.path.join("..", "1.download-data", "data")
+
+output_data_dir = "data"
+os.makedirs(output_data_dir, exist_ok=True)
+
+result_dir = "results"
+os.makedirs(result_dir, exist_ok=True)
+
+train_file = os.path.join(data_dir, "training_data.csv")
+
+# Load full data set
+full_train_df = pd.read_csv(train_file)
+print(f"Shape of full data set {full_train_df.shape}")
+
+train_df, test_df = train_test_split(
+    full_train_df, test_size=0.1, stratify=full_train_df.target
+)
+
+# Save trainings set
+file = os.path.join(output_data_dir, "train.tsv.gz")
```

Comment:
avoid using `file` as a variable name; use `filepath` instead.

Reply:
Remove whole file. This is replaced with the notebooks in the same folder

Reply:
Remove whole file. This is replaced with the notebooks in the same folder